tests/python_client/testcases/indexes/test_hnsw_prq.py

```
@@ -0,0 +1,258 @@
+import logging
+from utils.util_pymilvus import *
```

Ruff (0.14.10)

`from utils.util_pymilvus import *` used; unable to detect undefined names

(F403)